packages/fx-core/src/question/create.ts

```diff
@@ -124,7 +124,9 @@ export function projectTypeQuestion(): SingleSelectQuestion {
           return [projectType];
         }
       } else if (getRuntime(inputs) === RuntimeOptions.NodeJS().id) {
-        if (featureFlagManager.getBooleanValue(FeatureFlags.OfficeAddin)) {
+        if (featureFlagManager.getBooleanValue(FeatureFlags.OfficeMetaOS)) {
+          staticOptions.push(ProjectTypeOptions.officeMetaOS(inputs.platform));
+        } else if (featureFlagManager.getBooleanValue(FeatureFlags.OfficeAddin)) {
```

Comment:
It looks like the FeatureFlags.OfficeMetaOS is replacing FeatureFlags.OfficeAddin, is it necessary to keep the office addin feature flag?

Reply:
Not replace. The two feature gate control for two different create question options.